lib/core/device.rb

```diff
@@ -250,34 +250,28 @@ def close_app(action = nil)
   # launches the app specified by the Android app package / iOS bundle ID
   # defaults to the app under test if Value is not provided
   # Accepts:
-  #   Value (optional)
+  #   Value
   def launch_app(action)
+    raise "Application identifier is required!" unless action.key?("Value")
```

Comment:
Maybe we can say also that it is requiered to be writen specificly in the `Value` field?

Reply:
Done